snippets/numberWithCommas.md

```diff
@@ -0,0 +1,19 @@
+### numberWithCommas
+
+Use commas to separate digits。
+
+Use `toString()` to convert the number `num` to a string, then use `/\B(?=(\d{3})+(?!\d))/g` to separate every three characters with a comma,except word boundary。
```

Comment:
And change this sentence to:

Use `toString()` to convert the number `num` to a string, then use regex to separate every three characters of the integer part with a comma.